### DISCUSSION TITLE
Stress data

### DISCUSSION THREAD

**Discussion by atusiniida** [tcgoetz/GarminDB#172] [Q&A]
Dear colleagues,

I get stress data from garmin.db.
but I found that  the data looks different the stress data that we can see on the garmin connect website.  
In the data  from garmin.db, stress  levels get higher when we sleep, which is an opsite trend to those on the garmin connect website.
Can you have an explanation about this discrepancy?
the data  from garmin.db also contain minus values. 
what do the minus values mean?

Thank you in advance.

Best,
Atushi

**Comment by tcgoetz** [comment]
Stress data in Garmin DB is collected from Fit files downloaded from Garmin Connect. If GC is showing something different, then they must be post processing the data somehow.

Here's where the data is parsed from the Fit file and written to the database: https://github.com/tcgoetz/GarminDB/blob/master/garmindb/fit_file_processor.py#L162

**Comment by atusiniida** [comment]
I found stress data is off by 9 hours. scince I am warking on JST time, it is possible that the time setting causes the problem.
